translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,2502 @@
+"\n"
+"\t\t# Cria um serviço para um replication controller identificado por tipo e "
+"com o nome especificado em “nginx-controller.yaml”, que escuta na porta 80 e "
+"conecta na porta 8000 dos containers.\n"
+"\t\tkubectl expose -f nginx-controller.yaml —port=80 —target-port=8000\n"
+"\n"
+"\t\t# Cria um serviço para um pod valid-pod, que escuta na porta 444 com o "
+"nome “frontend”\n"
+"\t\tkubectl expose pod valid-pod —port=444 —name=frontend\n"
+"\n"
+"\t\t# Cria um segundo serviço baseado no serviço acima, expondo a Port 8443 "
+"do container como porta 443 e com nome “nginx-https”\n"
+"\t\tkubectl expose service nginx —port=443 —target-port=8443 —name=nginx-"
+"https\n"
+"\n"
+"\t\t# Cria um serviço para uma aplicação streaming replicada na porta 4100 "
+"com trafico balanceado UDP e nome ‘video-stream’.\n"
+"\t\tkubectl expose rc streamer —port=4100 —protocol=udp —name=video-stream\n"
+"\n"
+"\t\t# Cria um serviço para um nginx replicado using o replica set, que "
+"escuta na porta 80 e conecta na porta 8000 dos containers.\n"
+"\t\tkubectl expose rs nginx —port=80 —target-port=8000\n"
+"\n"
+"\t\t# Cria um serviço para um deployment nginx, que escuta na porta 80 e "
+"conecta na porta 8000 dos containers.\n"
+"\t\tkubectl expose deployment nginx —port=80 —target-port=8000"
+
+#: pkg/kubectl/cmd/delete.go:68
+msgid ""
+"\n"
+"\t\t# Delete a pod using the type and name specified in pod.json.\n"
+"\t\tkubectl delete -f ./pod.json\n"
+"\n"
+"\t\t# Delete a pod based on the type and name in the JSON passed into "
+"stdin.\n"
+"\t\tcat pod.json | kubectl delete -f -\n"
+"\n"
+"\t\t# Delete pods and services with same names \"baz\" and \"foo\"\n"
+"\t\tkubectl delete pod,service baz foo\n"
+"\n"
+"\t\t# Delete pods and services with label name=myLabel.\n"
+"\t\tkubectl delete pods,services -l name=myLabel\n"
+"\n"
+"\t\t# Delete a pod with minimal delay\n"
+"\t\tkubectl delete pod foo --now\n"
+"\n"
+"\t\t# Force delete a pod on a dead node\n"
+"\t\tkubectl delete pod foo --grace-period=0 --force\n"
+"\n"
+"\t\t# Delete all pods\n"
+"\t\tkubectl delete pods --all"
+msgstr ""
+"\n"
+"\t\t# Remove um pod usando o tipo e nome especificado no pod.json.\n"
+"\t\tkubectl delete -f ./pod.json\n"
+"\n"
+"\t\t# Remove um pod baseado no tipo e nome no JSON passado na entrada de "
+"comando(stdin).\n"
+"\t\tcat pod.json | kubectl delete -f -\n"
+"\n"
+"\t\t# Remove pods e serviços com os nomes “baz” and “foo”\n"
+"\t\tkubectl delete pod,service baz foo\n"
+"\n"
+"\t\t# Remove pods e serviços com label name=myLabel.\n"
+"\t\tkubectl delete pods,services -l name=myLabel\n"
+"\n"
+"\t\t# Remove um pod com um mínimo de delay\n"
+"\t\tkubectl delete pod foo —now\n"
+"\n"
+"\t\t# Força a remoção de  um pod em um node morto\n"
+"\t\tkubectl delete pod foo —grace-period=0 —force\n"
+"\n"
+"\t\t# Remove todos os pods\n"
+"\t\tkubectl delete pods —all"
+
+#: pkg/kubectl/cmd/describe.go:54
+msgid ""
+"\n"
+"\t\t# Describe a node\n"
```

Comment:
Hi @cpanato I notice there are still a few translations missing starting from this line 439

Reply:
Yes @femrtnz I need to finish that. I will try for this week

Reply:
60 more to complete